README.md

```
@@ -7,6 +7,7 @@ High‑quality experimental patterns & decorators built on top of `IMemoryCache`
 - [Components Overview](#components-overview)
 - [Quick Start](#quick-start)
 - [MeteredMemoryCache](#meteredmemorycache)
+- [OptimizedMeteredMemoryCache](#optimizedmeteredmemorycache)
 - [Implementation Details](#implementation-details)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)